worlds/hylics2/Options.py

```diff
@@ -1,25 +1,38 @@
 from dataclasses import dataclass
 from Options import Choice, Removed, Toggle, DefaultOnToggle, DeathLink, PerGameCommonOptions
 
+
 class PartyShuffle(Toggle):
-    """Shuffles party members into the pool.
-    Note that enabling this can potentially increase both the difficulty and length of a run."""
+    """
+    Shuffles party members into the pool.
+    
+    Note that enabling this can significantly increase both the difficulty and length of a run.
+    """
     display_name = "Shuffle Party Members"
 
+
 class GestureShuffle(Choice):
-    """Choose where gestures will appear in the item pool."""
+    """
+    Choose where gestures will appear in the item pool.
```

Comment:
This is such a tiny nitpick, but sometimes the option says "the pool" and sometimes it says "the item pool", so you may want to fix that up, or you can leave it alone, up to you.